gometalinter/def.bzl

```diff
@@ -8,26 +9,30 @@ def _gometalinter_impl(ctx):
         args.append("--no-config")
     args.extend(ctx.attr.paths)
     out_file = ctx.actions.declare_file(ctx.label.name + ".bash")
+    sdk = ctx.attr._go_sdk[GoSDK]
     substitutions = {
+        "@@GOMETALINTER_SHORT_PATH@@": shell.quote(ctx.executable._gometalinter.short_path),
+        "@@ARGS@@": shell.array_literal(args),
         "@@PREFIX_DIR_PATH@@": shell.quote(paths.dirname(ctx.attr.prefix)),
         "@@PREFIX_BASE_NAME@@": shell.quote(paths.basename(ctx.attr.prefix)),
-        "@@ARGS@@": shell.array_literal(args),
-        "@@GOMETALINTER_SHORT_PATH@@": shell.quote(ctx.executable._gometalinter.short_path),
+        "@@NEW_GOROOT@@": shell.quote(sdk.root_file.dirname),
```

Comment:
Is order meaningful here? I'm just curious why these lines moved around

Reply:
Order does not matter, I just made it consistent with the contents of the bash script template.